etl/src/main/java/org/pih/hivmigration/etl/sql/StagingDataMigrator.groovy

```diff
@@ -9,107 +9,121 @@ class StagingDataMigrator extends SqlMigrator {
                 openmrs_id int NOT NULL
             );
             
+            # Unknown location
             insert into hivmigration_health_center(hiv_emr_id, openmrs_id)
```

Comment:
Switched all these to be uuid-based, to protect against changing names breaking (siliently) our migration.